doc/api/fs.md

```diff
@@ -1662,7 +1666,9 @@ Note that it is unsafe to use `fs.writeFile` multiple times on the same file
 without waiting for the callback. For this scenario,
 `fs.createWriteStream` is strongly recommended.
 
-_Note: Specified file descriptors will not be closed automatically._
+_Note: If a file descriptor is specified as the `file`, it will not be closed 
+automatically. If a filename is specified as the `file`, the internally opened 
+file descriptor will be closed automatically._
```

Comment:
I’d be pretty fine with dropping it, too. The first sentence seems clear enough to me.